### PR TITLE
Update Proxy servers and Internal proxy servers

### PR DESCRIPTION
**Pull request by joinimran**
Made changes in Proxy servers and Internal proxy servers.
Problem:  https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9499

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md [hunk 1]
```diff
@@ -497,8 +497,7 @@ proxy.contoso.com:80;proxy2.contoso.com:443
 
 Specify the internal proxy servers your devices will go through to reach your cloud resources. Using this server type indicates that the cloud resources you’re connecting to are enterprise resources.
 
-This list shouldn’t include any servers listed in your Proxy servers list. 
-Proxy servers must be used only for non-WIP-protected (non-enterprise) traffic.
+This list shouldn’t include any servers listed in your Proxy servers list. Internal proxy servers must be used only for WIP-protected (enterprise) traffic.
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md:500]: Could we have the same amount of line numbers, both for easier comparison and also maybe to get a better source highlighting of the changes here on GitHub?
```suggestion
This list shouldn’t include any servers listed in your Proxy servers list.
Internal proxy servers must be used only for WIP-protected (enterprise) traffic.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me, both the new changes and the current source view for review purposes.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**